database/src/main/kotlin/no/nav/su/se/bakover/database/vedtak/snapshot/VedtakssnapshotJson.kt

```diff
@@ -125,7 +125,9 @@ internal sealed class VedtakssnapshotJson {
                     saksbehandler = saksbehandler.toString(),
                     attestering = attesteringer.hentSisteAttestering(),
                     oppgaveId = oppgaveId.toString(),
+                    // TODO ia/jm: Denne vil alltid være null når journalføring/distribusjon blir flyttet til asynkron jobb. Fjern?
```

Comment:
Enig. Kan evt. flytte den ut av konstruktøren og sette den statisk i kroppen. Også får vi vurdere om den har livets rett lenger og i så fall om vi bør oppdatere disse feltene senere i den asynke-jobben.

Reply:
Så vel for meg at vi bare kunne kvittet oss med disse feltene på denne typen - kunne sannsynligvis bare kvittet oss med typene på vedtak alltogheter og la distribusjonsobjektet forvalte dem videre.